scss/detail.scss

```diff
@@ -133,6 +133,12 @@
   justify-content: center;
 }
 
+.default-rectangle {
+  border-radius: 5px;
+  height: 32px;
+  width: 252px;
```

Comment:
I think the inputs should not be fixed width. Always try to use relative measures for the contents.
For example, if you change the width of the login box, will you also have to change the width of the inputs.